src/state/wallets/selectors.ts

```diff
@@ -76,6 +76,10 @@ export const walletsWithRecoveryTransaction = createSelector(wallets, walletsLis
   walletsList.filter(wallet => [HDSegwitP2SHArWallet.type, HDSegwitP2SHAirWallet.type].includes(wallet.type)),
 );
 
+export const subscribableWallets = createSelector(wallets, walletsList =>
```

Comment:
all wallets are subscribable

Reply:
Not yet. And also when trying to subscribe standard wallet and a valid 3keys wallet you will get an error for both of them. Let's allow to subscribe other than 3keys wallets when the backend is ready for that.

Reply:
so please leave the comment at least

Reply:
done